sdk/cosmos/azure-cosmos-kafka-connect/src/main/java/com/azure/cosmos/kafka/connect/implementation/CosmosExceptionsHelper.java

```diff
@@ -8,7 +8,7 @@
 import org.apache.kafka.connect.errors.ConnectException;
 import org.apache.kafka.connect.errors.RetriableException;
 
-public class KafkaCosmosExceptionsHelper {
+public class CosmosExceptionsHelper {
```

Comment:
Same here, there is another class in ImplementationBridgeHelper with the same name, it would be better to keep these classes with kafka naming. So that we can identify more clearly from any error stack trace.

Reply:
updated